src/app/components/FacetSidebar/FacetSidebar.jsx

```diff
@@ -212,6 +212,11 @@ FacetSidebar.propTypes = {
   keywords: React.PropTypes.string,
   selectedFacets: React.PropTypes.object,
   sortBy: React.PropTypes.string,
+  className: React.PropTypes.string,
+};
+
+FacetSidebar.defaultProps = {
```

Comment:
If the className is an empty string by default, I don't think you need to initialize it. Ensuring the PropType.string should be enough. If you don't pass in props.className, nothing will get rendered.